apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt

```diff
@@ -82,7 +82,7 @@ class OperationTypeSpecBuilder(
         .addAnnotation(Annotations.OVERRIDE)
         .addModifiers(Modifier.PUBLIC)
         .returns(queryFieldClassName)
-        .addStatement("return $VARIABLES_FIELD_NAME")
+        .addStatement("return ${VARIABLES_FIELD_NAME}")
```

Comment:
This is a quick fix suggested by the IDE, the curly braces are not needed here. It's totally unrelated to the PR, so I'm happy to revert this if you dont like it